README.md

```diff
@@ -1,2 +1,107 @@
-# eslint-udemy
-ESLint configs and plugins used by Udemy.
+Udemy ESLint Packages
```

Comment:
Should This have a # in front of it instead of = under it?

Reply:
Same comment for the other files.

Reply:
`=====` creates a page title, vs. `#` is 1st level header. I prefer `====`.

Reply:
Ah, thanks for teaching me 👍